src/odemis/acq/fastem.py

```diff
@@ -766,6 +767,11 @@ def get_abs_stage_movement(self):
         rel_move_hor = self.field_idx[0] * px_size[0] * field_res[0] * (1 - self._roa.overlap)  # in meter
         rel_move_vert = self.field_idx[1] * px_size[1] * field_res[1] * (1 - self._roa.overlap)  # in meter
 
+        # Acceleration unknown, therefore odemis.util.driver.estimateMoveDuration cannot be used to estimate the time.
+        estimated_time_x = guessActuatorMoveDuration(self._stage, "x", rel_move_hor)  # s
```

Comment:
I get the below ERROR when acquiring a ROA

```Traceback (most recent call last):
  File "/home/patel/development/odemis/src/odemis/gui/cont/fastem_acq.py", line 550, in on_acquisition_done
    future.result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 437, in result
    return self.__get_result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 389, in __get_result
    raise self._exception
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/patel/development/odemis/src/odemis/acq/fastem.py", line 600, in run
    self.pre_calibrate(self._pre_calibrations)
  File "/home/patel/development/odemis/src/odemis/acq/fastem.py", line 720, in pre_calibrate
    stage_pos = self.get_abs_stage_movement()
  File "/home/patel/development/odemis/src/odemis/acq/fastem.py", line 807, in get_abs_stage_movement
    estimated_time_x = guessActuatorMoveDuration(self._stage, "x", rel_move_hor)  # s
  File "/home/patel/development/odemis/src/odemis/util/driver.py", line 221, in guessActuatorMoveDuration
    return estimateMoveDuration(distance, speed, accel)
  File "/home/patel/development/odemis/src/odemis/util/driver.py", line 145, in estimateMoveDuration
    vp = math.sqrt(2.0 * distance / A)
ValueError: math domain error```

Reply:
sorry! I forgot to test the code :flushed:. I'll update the code